staging/src/kubevirt.io/client-go/go.mod

```diff
@@ -1,6 +1,8 @@
 module kubevirt.io/client-go
 
-go 1.21
+go 1.22.0
```

Comment:
Why?

Reply:
I think I saw a runtime log saying that it needed it.. will try to remove it back and see if I get it again.

Reply:
```
go: module ./staging/src/kubevirt.io/client-go requires go >= 1.22.0; switching to go1.22.2
```